logstash/tests/docker/logstash7/test_multi_pipelines.yml

```diff
@@ -1,5 +1,7 @@
 - pipeline.id: main
   path.config: "/usr/share/logstash/pipeline1"
+  queue.type: persisted
 
 - pipeline.id: second_pipeline
   path.config: "/usr/share/logstash/pipeline2"
+  queue.type: persisted
```

Comment:
Tests break if you set this to `memory` which is a valid pipeline setting. In terms of functional behaviour this is an extended test case we could test.